tools/drdeploy.c

```diff
@@ -520,7 +520,7 @@ expand_dr_root(const char *dr_root, bool debug, dr_platform_t dr_platform, bool
         { "lib32/debug/libdynamorio.dylib", true, true, false, DR_PLATFORM_32BIT },
         { "lib32/release/libdrpreload.dylib", false, false, true, DR_PLATFORM_32BIT },
         { "lib32/release/libdynamorio.dylib", true, false, false, DR_PLATFORM_32BIT },
-        { "lib64/debug/libdrpreload.dylib", true, false, true, DR_PLATFORM_64BIT },
+        { "lib64/debug/libdrpreload.dylib", true, true, true, DR_PLATFORM_64BIT },
```

Comment:
Please add more details to the PR description to give context to a reviewer: what observable and unintended behavior does this "fix"? Why is it justified to tag libdrpreload.dylib as `is_debug`?

Generally the tagged issue gives more context, but in this case I see that it is a generic MacOS-on-AArch64 support bug (which is totally fine), so the PR description should bridge the context gap for a reader.

Reply:
@ndrewh the very first PR comment is what turns into the commit message, so you have to edit that rather than adding a new comment.

Reply:
So this was a simple bug where this /debug/ path was marked release instead of debug.  If you could make that more clear as the current second comment was not clear to me when I read it.

Reply:
Please update PR description before merging.

Reply:
> So with this fix we will not try to look for libdrpreload anymore?

You can see on line 525 a /release/ path for libdrpreload.